pkg/apis/operator/v1alpha1/additional_options.go

```diff
@@ -25,9 +25,10 @@ import (
 // additional options will be updated on the manifests
 // these values will be final
 type AdditionalOptions struct {
-	Disabled                 *bool                                            `json:"disabled,omitempty"`
-	ConfigMaps               map[string]corev1.ConfigMap                      `json:"configMaps,omitempty"`
-	Deployments              map[string]appsv1.Deployment                     `json:"deployments,omitempty"`
-	HorizontalPodAutoscalers map[string]autoscalingv2.HorizontalPodAutoscaler `json:"horizontalPodAutoscalers,omitempty"`
-	StatefulSets             map[string]appsv1.StatefulSet                    `json:"statefulSets,omitempty"`
+	Disabled                    *bool                                            `json:"disabled,omitempty"`
+	ConfigMaps                  map[string]corev1.ConfigMap                      `json:"configMaps,omitempty"`
+	Deployments                 map[string]appsv1.Deployment                     `json:"deployments,omitempty"`
+	HorizontalPodAutoscalers    map[string]autoscalingv2.HorizontalPodAutoscaler `json:"horizontalPodAutoscalers,omitempty"`
+	StatefulSets                map[string]appsv1.StatefulSet                    `json:"statefulSets,omitempty"`
+	WebhookConfigurationOptions map[string]*WebhookConfigurationOptions          `json:"webhookConfigurationOptions,omitempty"`
```

Comment:
Why do we need pointer of `WebhookConfigurationOptions`?

Reply:
noneed to that